src/R/Editor/Impl/Validation/Tagger/EditorErrorTagger.cs

```diff
@@ -28,6 +29,8 @@ namespace Microsoft.R.Editor.Validation.Tagger {
     /// As the stylesheet changes, this class keeps the list of syntax errors up to date.
     /// </summary>
     public class EditorErrorTagger : ITagger<IErrorTag>, IEditorTaskListItemSource {
+        private readonly ICoreShell _shell;
+
         [Import(AllowDefault = true)]
         internal IEditorTaskList TaskList { get; set; }
```

Comment:
Fixed

---
In reply to: [68841881](https://github.com/Microsoft/RTVS/pull/2022#discussion_r68841881) [](ancestors = 68841881)